### ISSUE TITLE
Goblet will not fill with Animal blood

### ISSUE THREAD

**Issue by perashim** [elucent/eidolon#84]
Currently have Curios and Wyrmroost installed alongside Eidolon. I've tried sacrificing an animal over the goblet, near the goblet, under the goblet, but it will not fill with blood so I cannot perform the Sacrifice Animal ritual.

I've tried pigs, cows, chickens,sheep, and horses, but the goblet will not fill. 

Altar currently has a lantern, goblet, warped fungus, and a straw idol.

**Comment by Alexthw46** [comment]
For Anyone ended up here looking for the solution to this problem (sorry for late response btw), the goblet will accept blood from killing breedable animals so the ones you listed are okay. The range is pretty good so no problem of positioning the goblet.

The common cause to this problem AND the Cauldron water problem is the video setting: "Fabulous Graphics". Eidolon has a Rendering option called BetterLayering that conflicts with this new vanilla video setting. It can be toggled off inside the Eidolon config but honestly it's better to just switch to Fancy graphics (or Fast).

TL;DR: Switch to Fancy or Fast and it should appear.

**Comment by 4norak** [comment]
> For Anyone ended up here looking for the solution to this problem (sorry for late response btw), the goblet will accept blood from killing breedable animals so the ones you listed are okay. The range is pretty good so no problem of positioning the goblet.
> 
> The common cause to this problem AND the Cauldron water problem is the video setting: "Fabulous Graphics". Eidolon has a Rendering option called BetterLayering that conflicts with this new vanilla video setting. It can be toggled off inside the Eidolon config but honestly it's better to just switch to Fancy graphics (or Fast).
> 
> TL;DR: Switch to Fancy or Fast and it should appear.

Doesn't work for me. I tried fancy and fast but the blood was never displayed. The ritual worked, though, so the goblet was definitely filled
